sigal/gallery.py

```diff
@@ -232,6 +234,10 @@ def __init__(self, path, settings, dirnames, filenames, gallery):
         self.subdirs = dirnames
         self.output_file = settings['output_filename']
         self._thumbnail = None
+        
+        self.language = self.DEFAULT_LANGUAGE
+        if locale.getlocale():
+            self.language = locale.getlocale()[0][0:2]
```

Comment:
Really don't like that. I would appreciate a better solution.

Reply:
This could simply be a setting. I would prefer this instead of relying on locale, users may want to customize it.

Reply:
@saimn I thought it was a good idea to relay on locale to avoid too many settings (for instance, dates are formatting using locale). 

I will add a setting. Is ```language``` setting is enough precise or have you got a more relevant idea to name this new setting ?

Reply:
Yes it's once more setting, but I think it's good to be able to choose this language, same as for the `locale` setting. 

> Is language setting is enough precise 

Not sure, it may give the feeling that the theme can be translated. I would probably go for a more specific name, maybe `html_lang` or `html_language`?